### ISSUE TITLE
Slider makes webpage bounce on mobile devices.

### ISSUE THREAD

**Issue by TheBotox** [BlackrockDigital/startbootstrap-full-slider#4]
Hello, the slider makes the page bounce on mobile devices, because it forces the ratio and the dimensions of the images put into it. Can you solve it?

**Comment by davidtmiller** [comment]
Not sure on this one, I am having trouble replicating the bug on mobile. What sort of bounce are you talking about?

**Comment by vishnurb** [comment]
Hi, I am experiencing the same bouncing problem.
When I scroll down or up the page and pauses, it causes a jerk. It happens only once if I am scrolling in one direction (either upwards or downwards). It happens every time if I do scroll down -> pause -> scroll up ->pause. 
Hope this gives you enough clarity on the bug.

**Comment by davidtmiller** [comment]
I am hoping this gets fixed up in the Bootstrap 4 build coming soon. If the problem persists I will make a new issue.
